st2api/st2api/controllers/exp/root.py

```diff
@@ -23,3 +24,6 @@ class RootController(object):
     # Deprecated. Use /v1/ instead.
     actionalias = ActionAliasController()
     aliasexecution = ActionAliasExecutionController()
+
+    # Experimental
+    validators = ValidationController()
```

Comment:
Something about the /validators/mistral REST url bothers me - I am not able to exactly wrap my head around what is being created. It is definitely not a validator.

Since this is experimental I assume we have space to change the URL structure later. I guess I could get behind creating a `validation` where a `validation` is a noun similar to creation of an `actionexecution`